### ISSUE TITLE
Publishing of IATA One Record model as Linked Data?

### ISSUE THREAD

**Issue by jgmikael** [IATA-Cargo/ONE-Record#202]
Hi, 

any chance of publishing the IATA One Record Data Model as Linked Data with resolvable URIs and content negotiation on the server side? Now you have to download the offline Turtle file in order to use it in a modeling tool, am I right?

A reference model for this would be: https://vocabulary.uncefact.org/

**Comment by IATA-Cargo** [comment]
Hi,

Thank you very much for your suggestion. We are actively working on improving our documentation to make it more user-friendly.
If you want to navigate the IATA ONE Record data model you can use the following link: 
https://onerecord.iata.org/ns/cargo

Best regards,
Davide

**Comment by jgmikael** [comment]
Yes, navigating the html-pages works well, but it would be really cool to have a working link to a machine-readable format as well (like the JSON-LD?)

In the UN/CEFACT BSP JSON-LD Slack someone gave me this "short version" of what should be done:
"IATA will have to configure their http server to return partial or full machine readable vocabulary file upon receiving http request with "accept" header and url of the term belonging to their vocabulary"

Could this be done at some point?

**Comment by IATA-Cargo** [comment]
Hi,
At the top of the page you can fine the ontology in several format like JSON-LD, RDF/XML, N-Triples and TTL.

Best regards
Davide
